openwisp_radius/templates/custom_password_reset_email.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this email because you requested a password reset for your user account at
+{{ site_name }}.{% endblocktrans %}
+
+{% trans "Please go to the following page and choose a new password:" %}
+{% block reset_link %}
+{# Password reset confirm url #}
+{{password_reset_url}}/{{organization}}/password/reset/{{uid}}/{{token}}
```

Comment:
only `{{password_reset_url}}` must go here, and it must be calculated in the view